app/src/components/Layers/TracksLayer.js

```diff
@@ -103,6 +103,7 @@ export default class TracksLayerGL {
       // this.stage.drawCircle(circlePoints.inner.x[i], circlePoints.inner.y[i], circlePoints.inner.radius[i]);
       this.stage.drawCircle(circlePoints.inner.x[i], circlePoints.inner.y[i], 2);
     }
+    this.stage.endFill();
     // // inner range glow
     // this.stage.beginFill(color, 0.1);
```

Comment:
Is this code going to be used in the near future? If its old stuff, we should better remove it.